src/test-reader/mocha-reader/utils.js

```diff
@@ -59,6 +59,18 @@ const computeFile = mochaSuite => {
     return null;
 };
 
+const getMethodsByInterface = (mochaInterface = "bdd") => {
+    switch (mochaInterface) {
+        case "tdd":
```

Comment:
So there's no way to make this universal? =(

What if user defines its own interface? And I know for sure that we have users that use `exports` interface that are actively using Testplane

It's ok for MVP, but do we have a plan here?